Example/SettingsTableViewController.swift

```diff
@@ -77,13 +76,18 @@ class SettingsTableViewController: UITableViewController {
         }
     }
 
-    func saveCustomConfiguration() {
+    func saveCustomConfiguration(responseData: [MiniAppInfo]?) {
         self.save(field: self.textFieldAppID, for: .applicationIdentifier)
         self.save(field: self.textFieldSubKey, for: .subscriptionKey)
         self.displayAlert(title: NSLocalizedString("message_save_title", comment: ""),
             message: NSLocalizedString("message_save_text", comment: ""),
             autoDismiss: true) { _ in
                 self.dismiss(animated: true, completion: nil)
+                guard let miniAppList = responseData else {
+                    self.configUpdateDelegate?.settingsUpdated(controller: self, updated: nil)
+                    return
+                }
+                self.configUpdateDelegate?.settingsUpdated(controller: self, updated: miniAppList)
```

Comment:
We are doing this because from fetchAppList(), there is a scenario where we get status code as 200 with no mini-app list response. Please check fetchAppList()

Reply:
yeah I know, but `settingsUpdated` can manage a nil responseData as it's defined as optional :)